docs/Project.toml

```diff
@@ -1,5 +1,6 @@
 [deps]
+DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
```

Comment:
This should be added manually when instantiating (since it's a dev-dep; otherwise you might end with the wrong version).

```suggestion
```

Reply:
Ah, I didn't know that. Thanks!